VocaDbWeb/Code/DynamicImageUrlFactory.cs

```diff
@@ -1,5 +1,10 @@
 using System;
+#if NETFRAMEWORK

```

Comment:
I would actually recommend making a new project (VocaDbWeb.Shared) for these. It's complicated if VocaDbWeb.Core is referring to VocaDbWeb.